src/Middleware/HttpLogging/test/FileLoggerProcessorTests.cs

```diff
@@ -271,6 +271,70 @@ await using (var logger = new FileLoggerProcessor(new OptionsWrapperMonitor<W3CL
             }
         }
 
+        [Fact]
+        public async Task WritesToNewFileOnNewInstance()
```

Comment:
Shouldn't there be another test for changing the field options while the logger is still running?

Reply:
Added